crates/meroctl/src/cli/context/grant.rs

```diff
@@ -1,16 +1,34 @@
-use calimero_context_config::types::Capability;
+use calimero_context_config::types::Capability as ConfigCapability;
 use calimero_primitives::alias::Alias;
 use calimero_primitives::context::ContextId;
 use calimero_primitives::identity::PublicKey;
 use calimero_server_primitives::admin::{GrantPermissionRequest, GrantPermissionResponse};
-use clap::Parser;
+use clap::{Parser, ValueEnum};
 use eyre::OptionExt;
 use reqwest::Client;
 
 use crate::cli::Environment;
 use crate::common::{
     fetch_multiaddr, load_config, make_request, multiaddr_to_url, resolve_alias, RequestType,
 };
+use crate::output::Report;
+
+#[derive(Debug, Clone, ValueEnum, Copy)]
+pub enum Capability {
+    ManageApplication,
+    ManageMembers,
+    Proxy,
+}
```

Comment:
with no override, clap will default to `kebab-case` for these

https://docs.rs/clap/latest/clap/_derive/index.html#valueenum-attributes

which invalidates what's in the example report